app/javascript/App.jsx

```diff
@@ -5,10 +5,15 @@ import { Toaster } from 'react-hot-toast';
 import Header from './components/shared/Header';
 import { useAuth } from './contexts/auth/AuthProvider';
 import Footer from './components/shared/Footer';
+import useSiteSettings from './hooks/queries/admin/site_settings/useSiteSettings';
 
 export default function App() {
   const currentUser = useAuth();
   const containerHeight = currentUser?.signed_in ? 'full-height' : 'h-100';
+  const { data: siteSettings } = useSiteSettings();
```

Comment:
You should use `useSiteSetting` here and specify the name of the setting that you want - not `useSiteSettings`

Reply:
We will need to use PrimaryColorLight and PrimaryColorDark here as well - I'm not sure if that is what you mean?

Reply:
`useSiteSetting('PrimaryColor')`

Reply:
There already is one in the SiteSettings controller

Reply:
Maybe we can extend the controller to potentially take an array?

Reply:
The existing one only works for settings with true or false I believe ? He would need to change how it returns for it to work for all settings

Reply:
I have been using the Admin/SiteSettingsController - not the "regular" SiteSettingsController. I need to double check those.